iosHackthonApp/Modules/Features/AddMeal/Sources/AddMeal/AddMealView.swift

```diff
@@ -143,8 +91,143 @@ public struct AddMealView: View {
     }
 }
 
+// MARK:- Views
+private extension AddMealView {
+
+    struct InfoTextFieldsGroup: View {
+
+        @Binding var title: String
```

Comment:
Can these and the rest of the variables be private?

Reply:
I don't think these can be because the initialiser will match the access control of the properties, so if I make them private, the init becomes private.

But also the structs are private, and I don't keep a reference to the views anywhere, they are just created and used so I think it's safe

I have tried to get round it but haven't succeeded 😒